enos/provider/g5k.py

```diff
@@ -54,8 +54,16 @@ def init(self, conf, force_deploy=False):
         return (roles, network, (network_interface, external_interface))
 
     def destroy(self, env):
+
+        gridjob = None
         provider_conf = env['config']['provider']
-        gridjob, _ = EX5.planning.get_job_by_name(provider_conf['name'])
+
+        if 'jobid' in provider_conf:
```

Comment:
May I suggest to keep it in one line, and thus remove unnecessary `if` control flow e.g.,
```python
gridjob = provider_conf.get('jobid', EX5.planning.get_job_by_name(provider_conf['name'][0]))
```

Reply:
I've just modified these `if` in the second commit